glutton.go

```diff
@@ -2,26 +2,24 @@ package glutton
 
 import (
 	"io/ioutil"
+	"net"
 	"os"
 	"path/filepath"
+	"strings"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/kung-foo/freki"
+	"github.com/mushorg/glutton/out"
```

Comment:
We already have a log package (logrus) , May be I should rename this package to producer?